.github/workflows/lint.yaml

```diff
@@ -8,6 +8,15 @@ permissions:
   contents: read
 jobs:
   lint:
+    runs-on: ubuntu-latest
```

Comment:
We currently have two jobs that run go linting. Why two?

One job runs linting for all languages (go, sql) and any languages we might add in the future. 

The go lint job also lints go (using the same rules), but will add comments to the PR where the linting violations are.

Reason for both jobs linting go:
If someone is running linting locally, they can just run the lint docker container without needing to install golangci-lint. I feel making the golinter togglable in the container is unnecessarily right now (seems fine that the same linting rules are being run twice)

Reply:
<img width="1276" alt="image" src="https://github.com/stripe/pg-schema-diff/assets/130488190/91bfc854-1fe1-4568-aebf-cb88707bc8b4">

Example attached